src/main/java/core/basesyntax/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
```

Comment:
```suggestion
public class CsvReportServiceImpl implements ReportService {
```
this service has specific logic for csv format